internal/dev_server/ui/src/Switch.css

```diff
@@ -0,0 +1,77 @@
+
+.animated-switch-container {
```

Comment:
Going to be honest, most of this was written with Cursor.  It looks great to me, and functions perfectly, so not questioning it.  

It can be removed when we figure out the Switch issue in launchpad.